simplematrixbotlib/callbacks.py

```diff
@@ -59,8 +59,16 @@ async def invite_callback(self, room, event, tries=1):
             return
 
         try:
-            await self.async_client.join(room.room_id)
-            print(f"Joined {room.room_id}")
+            match = Match(room, event, self.bot)
+            if match.is_from_allowed_user():
+                await self.async_client.join(room.room_id)
+                print(f"Joined {room.room_id}")
+            else:
+                joined_rooms = await self.async_client.joined_rooms()
+                if room.room_id not in joined_rooms.rooms:
+                    # prevents leaving if already in the room opposed to rejecting invites
+                    await self.async_client.room_leave(room.room_id)
```

Comment:
An open question is whether actively rejecting, hence giving the inviter feedback and blocking further invites, is better than simply ignoring invites.

Reply:
I think we should ignore by default, but provide a config setting to allow rejection.